packages/animations/lib/src/shared_z_axis_transition.dart

```diff
@@ -345,11 +356,13 @@ class _EnterTransition extends StatelessWidget {
   final Animation<double> animation;
   final Widget child;
 
-  static Animatable<double> fadeInTransition = CurveTween(curve: decelerateEasing)
-    .chain(CurveTween(curve: const Interval(0.3, 1.0)));
+  static Animatable<double> fadeInTransition =
+      CurveTween(curve: decelerateEasing)
+          .chain(CurveTween(curve: const Interval(0.3, 1.0)));
 
-  static Animatable<double> scaleInTransition = Tween<double>(begin: 0.80, end: 1.00)
-    .chain(CurveTween(curve: standardEasing));
+  static Animatable<double> scaleInTransition =
```

Comment:
Two-space indent ... I think this would be a little more conventional formatting-wise (here and below):
```dart
  static Animatable<double> fadeOutTransition = FlippedCurveTween(curve: accelerateEasing)
    .chain(CurveTween(curve: const Interval(0.0, 0.3)));
```

Reply:
Done -- the formatted seemed to have removed my original formatting